pkg/fakerp/rotatesecrets.go

```diff
@@ -0,0 +1,48 @@
+package fakerp
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/openshift/openshift-azure/pkg/api"
+	"github.com/openshift/openshift-azure/pkg/plugin"
+)
+
+// handleRotateSecrets handles admin requests for the rotation of cluster secrets
+func (s *Server) handleRotateSecrets(w http.ResponseWriter, req *http.Request) {
```

Comment:
there's loads of copying of boilerplate here from handleRestore() - can you refactor so that it's DRY instead?

Reply:
I'll do this in a follow-up #1049